script/dump_templates

```diff
@@ -247,7 +255,22 @@ for my $table (keys %result) {
         }
         push @r, $r;
     }
-    $result{$table} = [@r];
+
+    if ($table eq 'Products') {
+        $result{$table} = [sort { productkey($a) cmp productkey($b) } @r];
+    }
+    elsif ($table eq 'JobTemplates') {
+        $result{$table} = [
+            sort {
+                productkey($a->{product})
+                  cmp productkey($b->{product} || $a->{testsuite}->{name} cmp $b->{testsuite}->{name})
```

Comment:
Seems like the `productkey($b->{product}` is missing closing ')'. It is only after `$b->{testsuite}->{name})` which I don't think is what was wanted.